provisioning/roles/kuma/files/vagrant/settings_local.py

```diff
@@ -3,6 +3,8 @@
 
 INTERNAL_IPS = ('127.0.0.1', '192.168.10.1',)
 
+ALLOWED_HOSTS = '*'
+
```

Comment:
That's interesting. Behavior doesn't match the documentation for me either. I get this error from runserver when `DEBUG = False` and there are no allowed hosts defined.

```
14:30:53 web.1        | CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
14:30:53 web.1        | exited with code 1
14:30:53 system       | sending SIGTERM to all processes
```